docs/rules/no-trailing-spaces.md

```diff
@@ -17,7 +19,7 @@ var foo = 0;//•••••
 var baz = 5;//••
```

Comment:
Suggest copy `//••••` from correct code for the option below to a line between the two variable declarations in incorrect code here.